produce/lib/produce/merchant.rb

```diff
@@ -0,0 +1,83 @@
+require 'spaceship'
+
+module Produce
+  class Merchant
+    def create(_options, _args)
+      login
+
+      merchant_identifier = Produce.config[:merchant_identifier] || UI.input("Merchant identifier (reverse-domain name style string starting with 'merchant'): ")
```

Comment:
Is this PR something people usually run on CI? If not, we might just leave it with a crash to not over-engineer?